CacheAdvance.podspec

```diff
@@ -1,13 +1,13 @@
 Pod::Spec.new do |s|
   s.name     = 'CacheAdvance'
-  s.version  = '1.0.1'
+  s.version  = '1.1.0'
   s.license  = 'Apache License, Version 2.0'
   s.summary  = 'A cache that enables the performant persistence of individual messages to disk'
   s.homepage = 'https://github.com/dfed/CacheAdvance'
   s.authors  = 'Dan Federman'
   s.source   = { :git => 'https://github.com/dfed/CacheAdvance.git', :tag => s.version }
   s.swift_version = '5.1'
-  s.source_files = 'Sources/CacheAdvance/**/*.{swift}', 'Sources/SwiftTryCatch/**/*.{h,m}'
+  s.source_files = 'Sources/**/*.{swift}', 'Sources/**/*.{h,m}'
```

Comment:
Should we make subspecs so that not everyone needs to pull down the ObjC wrapper if they don't need it?

Reply:
I'm tempted to avoid the complication of subspecs. The `CADCacheAdvance` code is pretty lightweight (won't add much to binary size), and it can be stripped by folk using static libraries who don't use it.

We can always improve by adding subspecs down the line, so we don't lose much by bundling everything together today.